code/modules/power/smes.dm

```diff
@@ -39,7 +39,7 @@
 
 /obj/machinery/power/battery/smes/initialize()
 	..()
-	spawn(5)
+	spawn(50)
```

Comment:
I made it spawn(5) with the earlier vault because it made the SMES work when testing on a local server, but it doesn't work half the time on the real server.